pkg/apis/garden/v1beta1/types.go

```diff
@@ -930,6 +930,8 @@ type GCPNetworks struct {
 	VPC *GCPVPC `json:"vpc,omitempty"`
 	// Workers is a list of CIDRs of worker subnets (private) to create (used for the VMs).
 	Workers []CIDR `json:"workers"`
+	// Internal is a private subnet (used for internal load balancers).
+	Internal []CIDR `json:"internal"`
```

Comment:
Your adding a new mandatory field here. What will happen to all the existing GCP shoots that have no specification for `internal` networks?

Reply:
Why is it a list of CIDRs if you later validate that only one value is inside?

Reply:
> Your adding a new mandatory field here. What will happen to all the existing GCP shoots that have no specification for `internal` networks?

Their value will be set to null, this could lead to a problem if the value is immutable, I will check it and change it accordingly.

Reply:
> Why is it a list of CIDRs if you later validate that only one value is inside?

To be persistent in the value definition of the shoot spec in regards to the aws-shoot. But we can change it, if you like it to be a single value.